### ISSUE TITLE
DB/Pools: fix pooling for mining nodes in Ghostlands

### ISSUE THREAD

**Issue by Wyreth** [TrinityCore/TrinityCore#20384]
**Description:**

Strangely enough, almost everything needed already exists in DB. Just the `pool_gameobject` entries are missing.

Closes #11743.

**Branch(es):** both (only checked on 3.3.5, though).

**TC rev. hash/commit:** 7a0f1d7

**TDB version:** 335.63

**Suggested fix**:
```sql
DELETE FROM `pool_gameobject` WHERE `guid` BETWEEN 75000 AND 75119;
INSERT INTO `pool_gameobject` (`guid`, `pool_entry`, `chance`, `description`) VALUES
(75000, 4700, 60, "Ghostlands Spawn Point 1 (Copper)"),
(75001, 4700, 30, "Ghostlands Spawn Point 1 (Tin)"),
(75002, 4700, 10, "Ghostlands Spawn Point 1 (Silver)"),
(75003, 4701, 60, "Ghostlands Spawn Point 2 (Copper)"),
(75004, 4701, 30, "Ghostlands Spawn Point 2 (Tin)"),
(75005, 4701, 10, "Ghostlands Spawn Point 2 (Silver)"),
(75006, 4702, 60, "Ghostlands Spawn Point 3 (Copper)"),
(75007, 4702, 30, "Ghostlands Spawn Point 3 (Tin)"),
(75008, 4702, 10, "Ghostlands Spawn Point 3 (Silver)"),
(75009, 4703, 60, "Ghostlands Spawn Point 4 (Copper)"),
(75010, 4703, 30, "Ghostlands Spawn Point 4 (Tin)"),
(75011, 4703, 10, "Ghostlands Spawn Point 4 (Silver)"),
(75012, 4704, 60, "Ghostlands Spawn Point 5 (Copper)"),
(75013, 4704, 30, "Ghostlands Spawn Point 5 (Tin)"),
(75014, 4704, 10, "Ghostlands Spawn Point 5 (Silver)"),
(75015, 4705, 60, "Ghostlands Spawn Point 6 (Copper)"),
(75016, 4705, 30, "Ghostlands Spawn Point 6 (Tin)"),
(75017, 4705, 10, "Ghostlands Spawn Point 6 (Silver)"),
(75018, 4706, 60, "Ghostlands Spawn Point 7 (Copper)"),
(75019, 4706, 30, "Ghostlands Spawn Point 7 (Tin)"),
(75020, 4706, 10, "Ghostlands Spawn Point 7 (Silver)"),
(75021, 4707, 60, "Ghostlands Spawn Point 8 (Copper)"),
(75022, 4707, 30, "Ghostlands Spawn Point 8 (Tin)"),
(75023, 4707, 10, "Ghostlands Spawn Point 8 (Silver)"),
(75024, 4708, 60, "Ghostlands Spawn Point 9 (Copper)"),
(75025, 4708, 30, "Ghostlands Spawn Point 9 (Tin)"),
(75026, 4708, 10, "Ghostlands Spawn Point 9 (Silver)"),
(75027, 4709, 60, "Ghostlands Spawn Point 10 (Copper)"),
(75028, 4709, 30, "Ghostlands Spawn Point 10 (Tin)"),
(75029, 4709, 10, "Ghostlands Spawn Point 10 (Silver)"),
(75030, 4710, 60, "Ghostlands Spawn Point 11 (Copper)"),
(75031, 4710, 30, "Ghostlands Spawn Point 11 (Tin)"),
(75032, 4710, 10, "Ghostlands Spawn Point 11 (Silver)"),
(75033, 4711, 60, "Ghostlands Spawn Point 12 (Copper)"),
(75034, 4711, 30, "Ghostlands Spawn Point 12 (Tin)"),
(75035, 4711, 10, "Ghostlands Spawn Point 12 (Silver)"),
(75036, 4712, 60, "Ghostlands Spawn Point 13 (Copper)"),
(75037, 4712, 30, "Ghostlands Spawn Point 13 (Tin)"),
(75038, 4712, 10, "Ghostlands Spawn Point 13 (Silver)"),
(75039, 4713, 60, "Ghostlands Spawn Point 14 (Copper)"),
(75040, 4713, 30, "Ghostlands Spawn Point 14 (Tin)"),
(75041, 4713, 10, "Ghostlands Spawn Point 14 (Silver)"),
(75042, 4714, 60, "Ghostlands Spawn Point 15 (Copper)"),
(75043, 4714, 30, "Ghostlands Spawn Point 15 (Tin)"),
(75044, 4714, 10, "Ghostlands Spawn Point 15 (Silver)"),
(75045, 4715, 60, "Ghostlands Spawn Point 16 (Copper)"),
(75046, 4715, 30, "Ghostlands Spawn Point 16 (Tin)"),
(75047, 4715, 10, "Ghostlands Spawn Point 16 (Silver)"),
(75048, 4716, 60, "Ghostlands Spawn Point 17 (Copper)"),
(75049, 4716, 30, "Ghostlands Spawn Point 17 (Tin)"),
(75050, 4716, 10, "Ghostlands Spawn Point 17 (Silver)"),
(75051, 4717, 60, "Ghostlands Spawn Point 18 (Copper)"),
(75052, 4717, 30, "Ghostlands Spawn Point 18 (Tin)"),
(75053, 4717, 10, "Ghostlands Spawn Point 18 (Silver)"),
(75054, 4718, 60, "Ghostlands Spawn Point 19 (Copper)"),
(75055, 4718, 30, "Ghostlands Spawn Point 19 (Tin)"),
(75056, 4718, 10, "Ghostlands Spawn Point 19 (Silver)"),
(75057, 4719, 60, "Ghostlands Spawn Point 20 (Copper)"),
(75058, 4719, 30, "Ghostlands Spawn Point 20 (Tin)"),
(75059, 4719, 10, "Ghostlands Spawn Point 20 (Silver)"),
(75060, 4720, 60, "Ghostlands Spawn Point 21 (Copper)"),
(75061, 4720, 30, "Ghostlands Spawn Point 21 (Tin)"),
(75062, 4720, 10, "Ghostlands Spawn Point 21 (Silver)"),
(75063, 4721, 60, "Ghostlands Spawn Point 22 (Copper)"),
(75064, 4721, 30, "Ghostlands Spawn Point 22 (Tin)"),
(75065, 4721, 10, "Ghostlands Spawn Point 22 (Silver)"),
(75066, 4722, 60, "Ghostlands Spawn Point 23 (Copper)"),
(75067, 4722, 30, "Ghostlands Spawn Point 23 (Tin)"),
(75068, 4722, 10, "Ghostlands Spawn Point 23 (Silver)"),
(75069, 4723, 60, "Ghostlands Spawn Point 24 (Copper)"),
(75070, 4723, 30, "Ghostlands Spawn Point 24 (Tin)"),
(75071, 4723, 10, "Ghostlands Spawn Point 24 (Silver)"),
(75072, 4724, 60, "Ghostlands Spawn Point 25 (Copper)"),
(75073, 4724, 30, "Ghostlands Spawn Point 25 (Tin)"),
(75074, 4724, 10, "Ghostlands Spawn Point 25 (Silver)"),
(75075, 4725, 60, "Ghostlands Spawn Point 26 (Copper)"),
(75076, 4725, 30, "Ghostlands Spawn Point 26 (Tin)"),
(75077, 4725, 10, "Ghostlands Spawn Point 26 (Silver)"),
(75078, 4726, 60, "Ghostlands Spawn Point 27 (Copper)"),
(75079, 4726, 30, "Ghostlands Spawn Point 27 (Tin)"),
(75080, 4726, 10, "Ghostlands Spawn Point 27 (Silver)"),
(75081, 4727, 60, "Ghostlands Spawn Point 28 (Copper)"),
(75082, 4727, 30, "Ghostlands Spawn Point 28 (Tin)"),
(75083, 4727, 10, "Ghostlands Spawn Point 28 (Silver)"),
(75084, 4728, 60, "Ghostlands Spawn Point 29 (Copper)"),
(75085, 4728, 30, "Ghostlands Spawn Point 29 (Tin)"),
(75086, 4728, 10, "Ghostlands Spawn Point 29 (Silver)"),
(75087, 4729, 60, "Ghostlands Spawn Point 30 (Copper)"),
(75088, 4729, 30, "Ghostlands Spawn Point 30 (Tin)"),
(75089, 4729, 10, "Ghostlands Spawn Point 30 (Silver)"),
(75090, 4730, 60, "Ghostlands Spawn Point 31 (Copper)"),
(75091, 4730, 30, "Ghostlands Spawn Point 31 (Tin)"),
(75092, 4730, 10, "Ghostlands Spawn Point 31 (Silver)"),
(75093, 4731, 60, "Ghostlands Spawn Point 32 (Copper)"),
(75094, 4731, 30, "Ghostlands Spawn Point 32 (Tin)"),
(75095, 4731, 10, "Ghostlands Spawn Point 32 (Silver)"),
(75096, 4732, 60, "Ghostlands Spawn Point 33 (Copper)"),
(75097, 4732, 30, "Ghostlands Spawn Point 33 (Tin)"),
(75098, 4732, 10, "Ghostlands Spawn Point 33 (Silver)"),
(75099, 4733, 60, "Ghostlands Spawn Point 34 (Copper)"),
(75100, 4733, 30, "Ghostlands Spawn Point 34 (Tin)"),
(75101, 4733, 10, "Ghostlands Spawn Point 34 (Silver)"),
(75102, 4734, 60, "Ghostlands Spawn Point 35 (Copper)"),
(75103, 4734, 30, "Ghostlands Spawn Point 35 (Tin)"),
(75104, 4734, 10, "Ghostlands Spawn Point 35 (Silver)"),
(75105, 4735, 60, "Ghostlands Spawn Point 36 (Copper)"),
(75106, 4735, 30, "Ghostlands Spawn Point 36 (Tin)"),
(75107, 4735, 10, "Ghostlands Spawn Point 36 (Silver)"),
(75108, 4736, 60, "Ghostlands Spawn Point 37 (Copper)"),
(75109, 4736, 30, "Ghostlands Spawn Point 37 (Tin)"),
(75110, 4736, 10, "Ghostlands Spawn Point 37 (Silver)"),
(75111, 4737, 60, "Ghostlands Spawn Point 38 (Copper)"),
(75112, 4737, 30, "Ghostlands Spawn Point 38 (Tin)"),
(75113, 4737, 10, "Ghostlands Spawn Point 38 (Silver)"),
(75114, 4738, 60, "Ghostlands Spawn Point 39 (Copper)"),
(75115, 4738, 30, "Ghostlands Spawn Point 39 (Tin)"),
(75116, 4738, 10, "Ghostlands Spawn Point 39 (Silver)"),
(75117, 4739, 60, "Ghostlands Spawn Point 40 (Copper)"),
(75118, 4739, 30, "Ghostlands Spawn Point 40 (Tin)"),
(75119, 4739, 10, "Ghostlands Spawn Point 40 (Silver)");
```

**Comment by velinath** [comment]
Just as a heads up, I think this would probably fix #11743 which has a bounty on it?

**Comment by Wyreth** [comment]
You're right, thanks :P Forgot to check the existing issues, again.
